File: app/src/main/java/org/lasque/effectcamerademo/views/cosmetic/OnItemClickListener.java
package org.lasque.effectcamerademo.views.cosmetic;

import androidx.recyclerview.widget.RecyclerView;

/**
 * TuSDK
 * org.lasque.tusdkdemohelper.tusdkpulse.newUI
 * qiniu-PLDroidMediaStreamingDemo
 *
 * @author H.ys
 * @Date 2020/8/5  14:31
 * @Copyright (c) 2020 tusdk.com. All rights reserved.
 */
public interface OnItemClickListener<I,H extends RecyclerView.ViewHolder> {
    void onItemClick(int pos, H holder, I item);
}

File: app/src/main/java/org/lasque/effectcamerademo/base/OnItemClickListener.kt
package org.lasque.effectcamerademo.base

import androidx.recyclerview.widget.RecyclerView

/**
 * TuSDK
 * org.lasque.tusdkeditoreasydemo.base
 * PulseDemo
 *
 * @author        H.ys
 * @Date        2020/10/22  11:54
 * @Copyright    (c) 2020 tusdk.com. All rights reserved.
 *
 */
interface OnItemClickListener<I,H : RecyclerView.ViewHolder> {
    fun onItemClick(pos: Int, holder: H, item: I)
}

interface OnItemDeleteClickListener<I,H : RecyclerView.ViewHolder>{
    fun onItemDelete(pos: Int, holder: H, item: I)
}